plugins/recommendations-plugin/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
     "compilerOptions": {
-      "target": "es6",
+      "target": "ES2017",
```

Comment:
We should really align the compiler options with the other plugins, in particular add "extends": "../../configs/base.tsconfig", There will be some errors compiling, but they are easily fixed.

Reply:
done within https://github.com/eclipse-che/che-theia/pull/1215, thanks Florent!